docs/app/pages/components/components-sections/badge/badge/badge.component.html

```diff
@@ -0,0 +1,69 @@
+<p>Demo here</p>
+
+<hr>
+
+<p>
+    The <code>ux-badge</code> directive can be used to easily attached a badge to any DOM element to be used as a label/tag, or to show (x) number of new items relating to it etc.
+</p>
+
+<p>The appearance and behavior of the directive can be configured using the following attributes:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="badgeBackgroundColor" type="string">
+        The color you want to use for the background color. This should be a string value that matches one of the colors available in the <code>ColorService</code>.
+    </tr>
+    <tr uxd-api-property name="badgeTextColor" type="string">
+        The color you want to use for the text/number on the badge. This should be a string value that matches one of the colors available in the <code>ColorService</code>. If you do not set this value the <code>ContrastService</code> will automatically pick which text color is suitable based on the <code>badgeBackgroundColor</code>.
+    </tr>
+    <tr uxd-api-property name="badgeText" type="string">
+        The text you wish to display on the badge. This property being set will trigger a check to detect if it is a number or text.
+    </tr>
+    <tr uxd-api-property name="badgeMaxLength" type="number">
+        This allows you to set a character / number limit on the text within the badge. If the content is number and larger than the specified size it will append a <strong>+</strong> otherwise it will the content as text and truncate the characters using ellipsis <strong>&hellip;</strong>.
+    </tr>
+    <tr uxd-api-property name="badgeAriaLabel" type="string">
+        The text you wish to used for accessibility devices, such as screen readers etc. If not set this will default to the <code>badgeText</code> value.
+    </tr>
+    <tr uxd-api-property name="badgeVerticalPosition" type="string" defaultValue="top">
```

Comment:
I don't see a 'center' variant in any of Roland's designs? Would it just be top and bottom? Alternatively we may want to follow Materials naming of 'above'/'below' and 'before' and 'after'?

Reply:
I'm guessing it was this:
![image](https://user-images.githubusercontent.com/26114101/78151816-e5bf6780-7430-11ea-9d85-9cc47304761d.png)

However I think that might be better as a separate setting or maybe a standalone inline component, since it has completely different spacing than the other alignments.

Reply:
Ah, I missed that. Yeah, I agree, might be good to be separate in the sense it is a inline element rather than something that is absolutely positioned

Reply:
I think I'll raise a separate ticket for that, we can discuss how to deliver it during story time.

Reply:
@ashley-hunter I added this myself - I thought there may be uses where you'd want it horizontally centered but I'll remove that. I'll follow the angular-material naming convention also.

Reply:
@donna-marijne yes I was trying to keep things cleaner by having inline as part of the vertical position options as they are related - I'll separate them as you have mentioned the difference in spacing.

Reply:
Here's the follow on ticket, for reference: https://portal.digitalsafe.net/browse/EL-3863